File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.cpp
class Solution {
public:
    int secondsBetweenTimes(string startTime, string endTime) {
        int starthr=stoi(startTime.substr(0,2));
        int startmin=stoi(startTime.substr(3,2));
        int startsec=stoi(startTime.substr(6,2));

         int endhr=stoi(endTime.substr(0,2));
        int endmin=stoi(endTime.substr(3,2));
        int endsec=stoi(endTime.substr(6,2));

        int st=(starthr*3600)+(startmin*60)+startsec;
        int end=(endhr*3600)+(endmin*60)+endsec;

        return end-st;
    }
};

File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.py
class Solution:
    def secondsBetweenTimes(self, sT: str, et: str) -> int:
        end=int(et[0:2])*3600+int(et[3:5])*60+int(et[6:])
        start=int(sT[0:2])*3600+int(sT[3:5])*60+int(sT[6:])
        return end-start